README.md

```diff
@@ -1,58 +1,53 @@
-
-## Compatibility
+6.  SSH into the machine, and run `sudo tailscale up` so you can
+    authenticate.
 
-The current package is confirmed to be working in different Synology models and architectures.
+## Building from source
 
-The package is created based on Tailscale [static binaries](https://pkgs.tailscale.com/stable/#static), and if your NAS has any of the supported architectures (x86, x86_64, arm, arm64) it should _just_ work.
+The source code for the Synology packages is kept in [Tailscale's main
+code repository](https://github.com/tailscale/tailscale). You can
+build the packages from source yourself with:
 
-If in doubt, check the [synology model list](docs/platforms.md) for the matching architecture.
-
-## Making packages
-
-This project hosts the machinery to build Tailscale Synology packages.
-
-You can build the packages using `make spkall` from the github.com/tailscale/tailscale repo
 ```bash
 git clone https://github.com/tailscale/tailscale.git
 cd tailscale
-make spkall
+go run ./cmd/dist build synology
 ```
-If everything worked you should have a directory called `spks` that contains your SPK files.
 
+If everything worked you should have a directory called `dist` that
+contains SPK files for all supported NASes and DSM versions.
+
+## Compatibility
+
+The package is confirmed to be working on various Synology models. For
+recent models, the correct package is usually the DSM7 package for
+`x86_64` or `armv8`. For older models based on 32-bit ARM, check the
+[synology model list](docs/platforms.md) to find the synology platform
+name.
 
 ## Credits and References
 
-- Thanks to [@nirev](https://github.com/nirev) for creating this project and transferring it to Tailscale's GitHub org.
-- https://haugene.github.io/docker-transmission-openvpn/synology-nas/ for the /dev/net/tun thing
-- Package structure: [Synology Package Developer Guide](https://help.synology.com/developer-guide/index.html)
-- Official Package building tool: [pkgscripts-ng](https://github.com/SynologyOpenSource/pkgscripts-ng)
-- The package building process was originally based on [synology-wireguard](https://github.com/runfalk/synology-wireguard) \
-If you need to _**compile**_ a synology package, check it out.
+- Thanks to [@nirev](https://github.com/nirev) for creating this
+  project and transferring it to Tailscale's GitHub org.
+- https://haugene.github.io/docker-transmission-openvpn/synology-nas/
+  for the /dev/net/tun thing
+- Package structure: [Synology Package Developer
+  Guide](https://help.synology.com/developer-guide/index.html)
+- The package building process was originally based on
+  [synology-wireguard](https://github.com/runfalk/synology-wireguard). If
+  you need to _**compile**_ a synology package, check it out.
```

Comment:
"If you need to _**compile**_ a synology package, check it out."
Probably not applicable any more.

Reply:
This is in the credits from the original author, citing the things they found useful. Reworded to clarify that it's handy for anyone who needs to cross-compile C code for Synology devices, unrelated to our own package.